news/_posts/core-weekly/2019-11-19-coreweekly-week-49-2019.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 49 of 2019"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2019-12-13
+authors: [ PrestaShop ]
+image: /assets/images/2017/04/core_weekly_banner.jpg
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 2nd to Sunday 8th of December 2019.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+## General messages
+
+Dear Developers,
+
+We were at [Paris Open Source Summit last week](https://www.opensourcesummit.paris/), and our developer advocate, Antoine Thomas, spoke about his [open source project activities and maturity dashboard](https://github.com/PrestaShop/open-source/tree/master/templates). We also are very glad to announce that we have won the [International Development Award](https://lesacteursdulibre.com/portfolio/prix-developpement-international/) at this event.
```

Comment:
@ttoine do you want to add something ? 😉

Reply:
Ok for me.